crates/bws/src/main.rs

```diff
@@ -461,7 +461,7 @@ async fn process_commands() -> Result<()> {
         } => {
             let secret = client
                 .secrets()
-                .create(&SecretCreateRequest {
+                .create(SecretCreateRequest {
```

Comment:
Having a somewhat hard time following, why did this change from a reference? We seem to be changing it shortly afterwards. Maybe something for a separate PR?

Reply:
Not sure why those were there, they aren't needed, I've removed them now